python/ray/tests/test_global_state.py

```diff
@@ -154,6 +155,43 @@ def ready(self):
     )
 
 
+def test_node_info_table(ray_start_cluster):
+    cluster = ray_start_cluster
+    cluster.add_node(node_name="head_node", include_dashboard=False)
+    # The _node_name here corresponds to the user's remote device
+    # since the user is bootstrapping from an existing cluster
+    # Is this behaviour that the user expects?
```

Comment:
maybe we should give a warning that the _node_name cannot be configured when bootstrap address is provided.

Reply:
we throw an error instead